planner/plan_traversal_test.go

```diff
@@ -0,0 +1,183 @@
+package planner
```

Comment:
Convention is that tests are placed in a separate package. Is there a reason for having them in the same package?

Reply:
No, no reason.  I did not realize that it was a convention.  I'll change it.